jenkins/toolbox-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -x
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
+
+# Run one of our tests inside a newly created toolbox container
+
```

Comment:
I would like to see here some test of number of command line argument, because we use them later in this bash script. When command line arguments are not used, then the script will produce confusing error messages.